pkgs/development/python-modules/NSKeyedUnArchiver/default.nix

```
@@ -0,0 +1,26 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "NSKeyedUnArchiver";
+  version = "1.2";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

and add `build-system = [ setuptools ];`